LASlib/src/LASlibConfig.cmake

```diff
@@ -0,0 +1,8 @@
+include("${CMAKE_CURRENT_LIST_DIR}/LASlibTargets.cmake")
+add_library(LAStools::LASlib INTERFACE IMPORTED)
+target_link_libraries(LAStools::LASlib INTERFACE LASlib)
+
+get_target_property(LASlib_INCLUDE_DIRS LASlib INTERFACE_INCLUDE_DIRECTORIES)
+set(LASlib_LIBRARIES LAStools::LASlib)
```

Comment:
This works, but... Using a target here make the target appear be in downstream transitive usage requirements. And targets require downstream packages to call `find_dependency`.

Reply:
I think the `find_dependency` requirements is a good practice to make the dependency clear?